### PR TITLE
Add basic UI

### PR DESCRIPTION
**Pull request by Mwapsam**
### OOP school library: add basic UI
**In this project, I created a form of UI for the school library app. This app can be invoked as an executable by running the following command in the terminal;**

`ruby app.rb`
### TOOLS USED

- _Ruby_
- _Interactive Ruby(irb)_
- _Terminal_

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+      puts 'Choose a number between 1 to 7'
+    end
+  end
+end
+
+def main
+  App.new
+end
+
+main
```
Comment (mmsesay) [inline app.rb]: - [ ] Please put the following lines of code in a separate file called `main.rb`  

- [ ] Secondly, the console app is not working as expected. Please see the test demo and the errors I encountered. I will suggest you work on fixing the errors, and verify that everything is working fine.

Test Demo|
:--------------:|
a1

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+  def list_rentals_by_person_id
+    print 'ID of person: '
+    id = gets.chomp
+
+    puts 'Rentals:'
+
+    @rentals.each { |rental| puts rental if rental.person.id == id.to_i }
+  end
+end
```
Comment (mmsesay) [inline app.rb]: - [OPTIONAL] Please put the following lines of code in a separate file. This will make reading easy.

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @Mwapsam, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+  def list_books
+    puts 'No books available' if @books.empty?
+    @books.each { |book| puts book }
+  end
+
+  def list_people
+    puts 'No people available' if @people.empty?
+    @people.each { |person| puts person }
+  end
```
Comment (rukundoeric) [inline app.rb]: - [ ] <b>REQUIRED:</b> Your listing is still not working as expected. It is listing class objects, not data. Kindly fix it.

scrnli_4_9_2022_2-53-57 AM

**Review by rukundoeric — APPROVED** [review]
Hi @Mwapsam, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app.rb (hunk 3 above)
Comment (Mwapsam) [reply app.rb]: Done!

**Review by addod19 — APPROVED** [review]
### Hi @Mwapsam, 

Your project is complete!  Nice job fixing the bug pointed out by the previous reviewer :1st_place_medal: 
There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Optional
- [ ] Kindly change how to run the app in the README file, it is `ruby app.rb` but since you have changed the entry point to main.rb, let us change it to `ruby main.rb`
- [ ] Also it would be better to always capitalize our commit messages

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**